contracts/test/Multisig.test.ts

```diff
@@ -780,3 +873,29 @@ describe("Multisig with multiple owners", function () {
     }
   });
 });
+
+const assertEvent = (
+  receipt: TransactionReceipt,
+  eventName: string,
+  eventData: IEventDataEntry[]
+) => {
+  const eventKey = getSelectorFromName(eventName);
+  const foundEvent = receipt.events.filter((e) =>
```

Comment:
Is the point here to actually find multiple events of the type or just one match? Could use .find() in that case.

Reply:
Well, this way we make sure there is only one such event emitted. "find" wouldn't care if there are multiple. But good observation.